src/accelerate/accelerator.py

```diff
@@ -2538,7 +2541,10 @@ def save_model(
 
         # Save the model
         for shard_file, shard in shards.items():
-            self.save(shard, os.path.join(save_directory, shard_file))
+            if safe_serialization:
+                safe_save_file(shard, os.path.join(save_directory, shard_file), metadata={"format": "pt"})
```

Comment:
Do we know for certain this only happens in `process_index==0`? Otherwise I'd rather see `self.save` be modified (which is really `utils.other.save`) to use `safe_serialize_file` instead.

Reply:
I guess it is safer to modify `self.save` instead. Modified !